src/tests/app.test.js

```diff
@@ -1,11 +1,38 @@
-const request = require('supertest')
-const app = require('../app')
+const request = require("supertest");
```

Comment:
To each their own, but ew.

Reply:
This is where prettier comes in. If you had a good config and a precommit hook that would fix this we wouldn't have these changes.

Reply:
Again, its prettier.  We need a `.prettierrc`